client/components/category-pill-navigation/index.tsx

```diff
@@ -70,6 +74,33 @@ export const CategoryPillNavigation = ( {
 		} );
 	}, [ selectedCategoryId ] );
 
+	if ( isMobile() ) {
```

Comment:
```suggestion
	const isMobile = useMobileBreakpoint();

	if ( isMobile ) {
```
This makes it reactive

Reply:
Honestly, I don't see necessity of using reactive approach, since I have never seen any customer who works with some website and decrease/increaser browser from desktop width to mobile 😅 And `useMobileBreakpoint` makes extra effort on browser, ofc it's very low, but anyway - if something is not useful then I don't see reason of using it. WDYT?

Reply:
> I have never seen any customer who works with some website and decrease/increaser browser from desktop width to mobile

Very uncommon, I agree, but all other aspects of the Pattern Library are responsive, and `useMobileBreakpoint` uses `window.matchMedia`, so I don't think performance is a realistic concern. This change also doesn't really add complexity, so I don't see a good reason not to use the hook

Reply:
I totally don't have objections :) I will update it